cloud/shared/variable_definitions.json

```diff
@@ -115,5 +115,11 @@
     "secret": false,
     "tfvar": true,
     "type": "bool"
+  },
+  "CIVIFORM_API_KEYS_BAN_GLOBAL_SUBNET": {
```

Comment:
Did you try deploying without providing this variable in config? What value is passed to fargate tasks, 'true'? Just want to make sure that empty string doesn't override the default true value of the terraform variable.

Reply:
verified that it's defaulting to true when not set